src/types/DataProvider.ts

```diff
@@ -1,15 +1,20 @@
 import { ajv } from '../ajv';
+import { KeycloakId, keycloakIdSchema } from './KeycloakUserId';
 import type { JSONSchemaType } from 'ajv';
-import type { Writable } from './Writable';
 import type { ShortCode } from './ShortCode';
 
 interface DataProvider {
 	readonly shortCode: ShortCode;
 	name: string;
+	keycloakOrganizationId?: KeycloakId | null;
 	readonly createdAt: string;
 }
 
-type WritableDataProvider = Writable<DataProvider>;
+// Using `Writable` here instead of explicit `Pick` messes up the optional field.
```

Comment:
I wrote a patch for this in another PR, just removed it and made it its own thing so we can merge it and benefit from it in this case / not depend on the other place. https://github.com/PhilanthropyDataCommons/service/pull/1378

Let's merge that and then you won't need to remove use of `Writable` here.